plugin/documents.py

```diff
@@ -308,7 +308,7 @@ def _update_diagnostic_in_status_bar_async(self) -> None:
         self.view.erase_status(self.ACTIVE_DIAGNOSTIC)
 
     def _update_inline_diagnostic_async(self) -> None:
-        region_key = "lsp_d-a"
+        region_key = "lsp_d-ann"
```

Comment:
Maybe be a more explicit about the region key... 
a -> ann -> annotations

I know that we have places in the code where we use one letter to describe the region key, but it takes time to understand what that means.
So i would suggest using a whole word over abbreviation.

Reply:
I guess my thinking was that I might save a lot of bytes in memory but since I'm only showing active diagnostic, it's never gonna be an issue.